packages/e2e-tests/fixtures/blocks/core__latest-posts.json

```diff
@@ -17,7 +17,8 @@
             "displayFeaturedImage": false,
             "featuredImageSizeSlug": "thumbnail",
             "featuredImageSizeWidth": null,
-            "featuredImageSizeHeight": null
+            "featuredImageSizeHeight": null,
+			"addLinkToFeaturedImage": false
```

Comment:
a weird indent  :)

Reply:
The previous json had `spaces`, so I changed them to `tabs`. Same as below..